src/RegistryCoordinator.sol

```diff
@@ -503,6 +511,24 @@ contract RegistryCoordinator is
         return operatorId;
     }
 
+    /**
+     * @notice Validates that an incoming operator is eligible to replace an existing
+     * operator based on the stake of both
+     * @dev In order to churn, the incoming operator needs to have more stake than the
+     * existing operator by a proportion given by `kickBIPsOfOperatorStake`
+     * @dev In order to be churned, the existing operator needs to have less stake than
+     * the total quorum stake by a propertion given by `kickBIPsOfTotalStake`
```

Comment:
typo in `propertion`
I think this wording may also be a little confusing?
I would say instead:
```
     * @dev In order to be churned out, the existing operator needs to have a proportion of the total
     * quorum stake less than `kickBIPsOfTotalStake`
```
but perhaps that's just my personal preference